introduction.rst

```diff
@@ -105,12 +106,40 @@ Conan is being used in production by thousands of companies like TomTom, Audi, R
 
 But an essential part of Conan is that many of those users will contribute back, creating an amazing and helpful community:
 
-- The https://github.com/conan-io/conan project has around 6K stars in Github and counts with contributions of almost 300 different users (this is just the client tool).
+- The https://github.com/conan-io/conan project has around 6.5K stars in Github and counts with contributions from more than 300 different users (this is just the client tool).
 - Many other users contribute recipes for ConanCenter via the https://github.com/conan-io/conan-center-index repo, creating packages for popular Open Source libraries, contributing many thousands of Pull Requests per year.
 - More than two thousands Conan users hang around the `CppLang Slack #conan channel <https://cppalliance.org/slack/>`_, and help responding to questions, discussing problems and approaches, making it one of the most active channels in the whole CppLang slack.
 - There is a Conan channel in `#include<cpp> discord <https://www.includecpp.org/discord/>`_.
 
 
+How to read this documentation
+------------------------------
+
+This documentation has very different sections:
+
+- The **tutorial** is an actual hands-on tutorial, with examples and real code, intended to be played sequentially from beginning to end,
+  running the exercises in your own computer. There is a conducting thread and exercises might depend on some previous explanations and code.
```

Comment:
What is a "conducting thread" ?

Reply:
Yes, sorry, I don't know how to say it in english.
I want to say this is a story, the tutorial implements developers stories, with incremental complexity, in a natural and continuous way, like a section is continuation of the previous one, and there is like a guiding narrative over the whole tutorial.

Reply:
Maybe this?

```suggestion
  running the exercises in your own computer. There is a "narrative" to this section and the exercises might depend on some previous explanations and code - building on the previous example.
```